examples/playbooks/lots_of_warnings.transformed.yml

```diff
@@ -0,0 +1,1000 @@
+---
```

Comment:
[This](https://github.com/cognifloyd/ansible-lint/blob/yaml-fmt/examples/playbooks/lots_of_warnings.transformed.yml) is a reformatted copy of [examples/playbooks/lots_of_warnings.yml](https://github.com/cognifloyd/ansible-lint/blob/yaml-fmt/examples/playbooks/lots_of_warnings.yml)